perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_userPreferredCertDN.java

```diff
@@ -87,8 +87,6 @@ public AttributeDefinition getAttributeDefinition() {
 		return attr;
 	}
 
-	//TODO what dependencies of this attribute???
-
```

Comment:
There is already dependency and I think it is correct "NS_USER_ATTR_DEF:userCertDNs".